apps/frontend/src/libs/components/loader/styles.module.css

```
@@ -0,0 +1,62 @@
+:root {
+	--size: 50px;
+	--weight: 5px;
+	--animation-time: 2s;
+}
```

Let's remove variables that are used only once; for animation-duration, it's better to keep this in .loader scope, not in the root
```suggestion
.loader {
	--animation-duration: 2s;
}
```